pkg/apis/submariner.io/v1/types.go

```diff
@@ -86,11 +86,30 @@ type GatewayStatus struct {
 // LatencySpec describes the round trip time information in nanoseconds for a packet
 // between the gateway pods of two clusters.
 type LatencySpec struct {
-	LastRTT    uint64 `json:"lastRTT"`
-	MinRTT     uint64 `json:"minRTT"`
-	AverageRTT uint64 `json:"averageRTT"`
-	MaxRTT     uint64 `json:"maxRTT"`
-	StdDevRTT  uint64 `json:"stddevRTT"`
+	// This shall be merged once the operator is using the latest.
+	// Using Optional to avoid validation errors when this field
+	// is not used.
+	// +optional
+	LastRTT uint64 `json:"lastRTT,omitempty"`
+	// +optional
+	MinRTT uint64 `json:"minRTT,omitempty"`
+	// +optional
+	AverageRTT uint64 `json:"averageRTT,omitempty"`
+	// +optional
+	MaxRTT uint64 `json:"maxRTT,omitempty"`
+	// +optional
+	StdDevRTT uint64 `json:"stddevRTT,omitempty"`
+
+	// +optional
+	LastRTTInNs uint64 `json:"lastRTTInNs,omitempty"`
```

Comment:
How about `Nanos` to be a bit more descriptive?

```suggestion
	LastRTTInNanos uint64 `json:"lastRTTInNs,omitempty"`
```

Reply:
Done.